sequencers/59902/all.json

```diff
@@ -133,5 +133,14 @@
     "seq_addr": "0x95C648c3Ae429B3B17F7fbA6ACa217824E7712A4",
     "pubkey": "0xa81ebea6581bc7f97c082bae2fdf950134341737e65fd0985a094f421830ebf9acb4793782f3490446123db1565a33e10c1ac17dfde22ebeba0869b766b8856d",
     "url": "https://nodies.org"
+  },
+  {
+    "name": "A41",
+    "avatar": "{BASEDIR}/assets/a41-logo.png",
+    "desc": "A41 is a blockchain infrastructure service provider that aims to lead the crypto industry and build solid foundations for all ecosystems.",
+    "address": "0x907cfca1398c363505FF8bE3F781FF102499dc23",
```

Comment:
If stakelink contract will initiate the lockup and will handle increasing and decreasing the stake, then this address must be the stakelink contract address, and not an EOA